### PR TITLE
ref(vsts): Changed the issue display name to include the project name

### PR DESCRIPTION
**Pull request by lauryndbrown**

### CODE REVIEW

**Comment by lauryndbrown** [comment]
screen shot 2018-08-15 at 12 40 48 pm 2

**Comment by lauryndbrown** [comment]
@saragilford is the above photo what you were looking for?

File: src/sentry/integrations/vsts/issues.py [hunk 1]
```diff
@@ -81,6 +81,7 @@ def create_issue(self, data, **kwargs):
         return {
             'key': created_item['id'],
             # 'url': created_item['_links']['html']['href'],
```
Comment (MeredithAnya) [inline src/sentry/integrations/vsts/issues.py]: Do we wanna remove this?

**Review by MeredithAnya — APPROVED** [review]

**Comment by saragilford** [comment]
@lauryndbrown yep, assuming `HelloWorld` is the project and not the account, works for me!

File: src/sentry/integrations/vsts/issues.py (hunk 1 above)
Comment (lauryndbrown) [reply src/sentry/integrations/vsts/issues.py]: Sure I can follow up with another PR. That's a bit less important and this needs to get out quickly. But this was from the original vsts plugin and I just copied over as I wasn't sure if we were going to use it.

**Comment by lauryndbrown** [comment]
@saragilford lol sorry! Yes. HelloWorld is the name of my project in VSTS

**Pull request merged**